core/reporter-impl/src/test/java/org/arquillian/reporter/impl/model/report/AllReportsMergeTest.java

```diff
@@ -17,6 +13,10 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
```

Comment:
Please, clean the PR so it doesn't contain any formatting changes in files that haven't been changed